docs/Product/AboutTheOETProduct.mdx

```diff
@@ -6,25 +6,27 @@ sidebar_label: "About the OET Product"
 import MaintainerTag from "@site/src/components/MaintainerTag";
 import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 
-<MaintainerTag maintainerEmails={["jane.smith@example.com", "john.doe@example.com"]} />
+<MaintainerTag maintainerEmails={["mario.dzamarija@openenergytransition.org", "ekaterina.fedotova@openenergytransition.org"]} />
 
 # About the OET Product
-this should be same as sidebar_label and folder name
 
 <ControlledDocBanner />
 
-should be present on every controlled document.
+Open Energy Transition continues to build on open source software. PyPSA project, initially started in Academia, back in 2016, is being developed actively.
 
-## Sub Heading
+As of now, the PyPSA engine counts approx. 25,000 lines of Python code.
 
-At our company, we are committed to providing a safe and inclusive workplace for all employees. This policy outlines our expectations for behavior and conduct in the workplace.
+## Product Development
 
-## Another Sub Heading
+Open Energy Transition actively uses, and extend PyPSA, for the purposes of studies, consultations, workshops, presentations, conferences and projects.
 
-All employees are expected to adhere to the following code of conduct:
+- Any such modifications are backfed upstream, to the original PyPSA-X repository.
 
-- Treat all colleagues with respect and professionalism
-- Avoid any behavior that could be considered discriminatory or harassing
-- Maintain confidentiality of company and client information
-- Follow all company policies and procedures
-- Report any violations of this code of conduct to management
+The code gets reviewed & modified by dozens of associates/contributors in power systems planning, operation & energy economics, many of whom are experts/PhDs in their specific areas.
```

Comment:
Please remove experts/PhDs. I am not an expert in any area related to power energy modelling, yet OET welcomes my contributions.

We do not want to scare people off :)

Reply:
You are right. I have not thought about this, and I modified, as you have done previously.

Please check the new sentence version in the modified commit of this PR.